### PR TITLE
docs(legal): add footnote and update footer DOC-362

### PR DESCRIPTION
**Pull request by 317brian**
Resolves Jira: [DOC-362]

Footnote: https://s.armory.io/RBuYYk4w
Footnote part 2 + footer: https://s.armory.io/8LubblkP
Deploy preview: https://deploy-preview-563--armory-docs.netlify.app/docs/armory-enterprise-matrix/

[DOC-362]: https://armory.atlassian.net/browse/DOC-362

### CODE REVIEW

**Comment by npetrell** [comment]
Brian, Can we alter the language in the proprietary footnote? Instead of saying "Some or all" just say "Some" as clearly some of the features on the page are listed as OSS which wouldn't need a license. Some can mean all anyways. Otherwise, looks good to me.

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -19,11 +19,11 @@
       <div class="col-12 col-sm-4 text-center py-2 order-sm-2">
         {{ with .Site.Params.copyright }}<small class="text-white">&copy; {{ now.Year}} {{ .}} {{ T "footer_all_rights_reserved" }}</small>{{ end }}
 		  <br>
-
-		  {{ with .Site.Params.terms_and_conditions }}<small class="ml-1"><a href="{{ . }}" target="_blank">{{ T "footer_terms_and_conditions" }}</a></small>{{ end }}
-
-		  {{ with .Site.Params.terms_of_service }}<small class="ml-1"><a href="{{ . }}" target="_blank">{{ T "footer_terms_of_service" }}</a></small>{{ end }}
-
+      <small class="text-white ml-1">Use of Armory products and features are subject to the
```
Comment (aimeeu) [inline layouts/partials/footer.html]: Small grammar nit: 
```suggestion
      <small class="text-white ml-1">Use of Armory products and features is subject to the
```
"Use" is singular...

File: content/en/docs/armory-enterprise-matrix.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ This page describes the features and capabilities that Armory supports. Note tha
 
 ![OSS](/images/oss.svg) The feature or parts of it are available in Open Source Spinnaker.
```
Comment (aimeeu) [inline content/en/docs/armory-enterprise-matrix.md]: open source Spinnaker vs Open Source Spinnaker ??

We seem to use "Open Source" and "open source" interchangeably. Google style guide lists it as "open source" in the [word list](https://developers.google.com/style/word-list#letter-o).

**Review by aimeeu — CHANGES_REQUESTED** [review]
Few nits

File: content/en/docs/armory-enterprise-matrix.md (hunk 2 above)
Comment (317brian) [inline content/en/docs/armory-enterprise-matrix.md]:
```suggestion
![OSS](/images/oss.svg) The feature or parts of it are available in open source Spinnaker.
```

**Review by aimeeu — APPROVED** [review]
lgtm - thanks!

**Pull request merged**